src/swaps/boltz.rs

```diff
@@ -748,9 +753,13 @@ pub enum WsRequest {
 
 impl WsRequest {
     pub fn subscribe_swap_request(swap_id: &str) -> Self {
+        Self::subscribe_swaps_request(vec![swap_id.to_string()])
```

Comment:
```suggestion
    pub fn subscribe_swap_request(swap_id: String) -> Self {
        Self::subscribe_swaps_request(vec![swap_id])
```
Would be nicer but is breaking